task/01-strings-tasks.js

```diff
@@ -145,7 +145,7 @@ function removeFirstOccurrences(str, value) {
  *   '<a>' => 'a'
  */
 function unbracketTag(str) {
-    throw new Error('Not implemented');
+    return str.slice(1,-1);
```

Comment:
According to the comment above, it would be better to add a space between function arguments